app/pages/admin/user-settings/delete-user.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+
+import LoadingIndicator from '../../../components/loading-indicator';
+
+class DeleteUser extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {deletionInProgress: false};
+    this.handleClick = this.handleClick.bind(this)
+    this.performDelete = this.performDelete.bind(this)
+  }
+
+  render() {
+    return (
+      <div>
+        <button type="button" className="minor-button" disabled={this.state.deletionInProgress} onClick={this.handleClick} style={{background: "red", color: "white"}}>
+          Delete this user
+          <LoadingIndicator off={!this.state.deletionInProgress} />
+        </button>
+        {this.state.deletionError && (
+           <div className="form-help error">{this.state.deletionError.message}</div>
+        )}
+      </div>
+    );
+  }
+
+  handleClick() {
+    this.setState({deletionError: null});
+
+    var phrase = prompt("You are about to delete this user! This cannot be reversed.\nEnter this user's login to confirm.");
```

Comment:
I'm not terribly excited about this solution, but on the other hand, it's only an admin tool?

Reply:
```
if (window.confirm(`Do you really want to delete ${this.props.user.login}?`) {
  this.performDelete();
}
```

Reply:
It's fine by me.

Reply:
Only thing perhaps, is there any ambiguity between user's `login` and `display_name`?

Reply:
Ah, I like Jim's idea too, which removes my concerns.

Reply:
Depends how difficult you want to make deleting a user, I guess.